app/views/policy_classes/_comment.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= content_tag(:div, data: { controller: "delete-record" }) do %>
```

Comment:
Most of this file is just `app/views/policy_classes/_comment_form.html.erb` renamed!